### PR TITLE
testing/xbps: new aport

### PR DESCRIPTION
**Pull request by misery**
Add package management of VoidLinux like
pacman or dpkg to bootstrap VoidLinux.

### CODE REVIEW

File: testing/xbps/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Contributor: André Klitzing <aklitzing@gmail.com>
+# Maintainer: André Klitzing <aklitzing@gmail.com>
+
+pkgname=xbps
+pkgver=0.51
+pkgrel=0
+pkgdesc="XBPS to bootstrap VoidLinux"
```
Comment (kaniini) [inline testing/xbps/APKBUILD]: the package description should be more accurate, while it is true that it is mainly meant for bootstrapping voidlinux, that doesn't necessarily have anything to do with the package itself

**Comment by misery** [comment]
Done :-)

File: testing/xbps/APKBUILD (hunk 1 above)
Comment (misery) [inline testing/xbps/APKBUILD]: done

**Comment by jirutka** [comment]
@misery Is this package complete and fully working? I’d like to move it into community.

**Comment by misery** [comment]
@jirutka Yes, it works without problems for me.

**Pull request closed without merging**